sql/core/src/test/scala/org/apache/spark/sql/DataFrameFunctionsSuite.scala

```diff
@@ -1785,6 +1785,13 @@ class DataFrameFunctionsSuite extends QueryTest with SharedSparkSession {
       ),
       queryContext = Array(ExpectedContext("", "", 0, 32, "array_contains('a string', 'foo')"))
     )
+
+    val schema = StructType(Seq(
+      StructField("a", ArrayType(IntegerType, containsNull = true)),
+      StructField("b", IntegerType)))
+    val data = Seq(Row(Seq[Integer](1, 2, 3, null), null))
+    val df1 = spark.createDataFrame(spark.sparkContext.parallelize(data), schema)
+    checkAnswer(df1.select(array_contains(col("a"), col("b"))), Seq(Row(true)))
```

Comment:
I'm not sure if this belongs to `break changes`, but I think it should be a correct correction.